pyfstat/make_sfts.py

```diff
@@ -334,8 +340,21 @@ def _get_setup_from_timestamps(self):
         self.sftfilenames = []  # This refers to the MFD output!
         for X, IFO in enumerate(IFOs):
             tsX = np.genfromtxt(tsfiles[X], comments="%")
```

Comment:
@dbkeitel this is going to read as a float unless we tell it otherwise.

If we force integer, we will not know what happened.

So... we cannot meaningfully check for float / integer.

Opinion?